vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -211,6 +212,26 @@ void save_action_scores(ccb& data, decision_scores_t& decision_scores)
   data.shared->pred.a_s.clear();
 }
 
+void remove_labeled_action(ccb& data, const multi_ex& examples)
```

Comment:
```suggestion
void add_chosen_action_to_exclude_list(ccb& data, const multi_ex& examples)
```

or something similar, remove implies that something is actually removed, i.e. that this method has side effects